extensions/cornerstone/src/Viewport/Overlays/ViewportOverlay.tsx

```diff
@@ -1,278 +0,0 @@
-import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
I don't believe this is used. However I could be wrong and I have no problem bringing it back if we should.

Reply:
I think you should put it back, this is used when customizationService decides to use it